tests/integration/modifiers/style-test.js

```diff
@@ -15,77 +15,108 @@ module('Integration | Modifiers | style', function(hooks) {
 
-    assert.dom('p').hasNoAttribute("style");
+    assert.dom('p').hasNoAttribute('style');
   });
 
   test('it supports dasherized property name', async function(assert) {
     await render(hbs`<p {{style font-size="6px"}}></p>`);
 
-    assert.dom('p').hasStyle({ fontSize: "6px" });
+    assert.dom('p').hasStyle({ fontSize: '6px' });
   });
 
   test('it supports camelCase property name', async function(assert) {
     await render(hbs`<p {{style fontSize="6px"}}></p>`);
 
-    assert.dom('p').hasStyle({ fontSize: "6px" });
+    assert.dom('p').hasStyle({ fontSize: '6px' });
   });
 
   test('it supports String object', async function(assert) {
-    this.set('display', new String("none"));
+    this.set('display', new String('none'));
     await render(hbs`<p {{style display=display}}></p>`);
 
-    assert.dom('p').hasStyle({ display: "none" });
+    assert.dom('p').hasStyle({ display: 'none' });
   });
 
   test('it observers values for changes', async function(assert) {
     this.set('display', 'none');
     await render(hbs`<p {{style display=display}}></p>`);
 
-    assert.dom('p').hasStyle({ display: "none" });
+    assert.dom('p').hasStyle({ display: 'none' });
 
     this.set('display', 'inline');
-    assert.dom('p').hasStyle({ display: "inline" });
+    assert.dom('p').hasStyle({ display: 'inline' });
   });
 
   module('options hash', function() {
     test('it accepts an option hash as alternative to named arguments', async function(assert) {
       await render(hbs`<p {{style (hash display="none")}}></p>`);
 
-      assert.dom('p').hasStyle({ display: "none" });
+      assert.dom('p').hasStyle({ display: 'none' });
     });
 
     test('options hash and named arguments could be used together', async function(assert) {
       await render(hbs`<p {{style (hash padding="6px") margin="12px"}}></p>`);
 
-      assert.dom('p').hasStyle({ padding: "6px", margin: "12px" });
+      assert.dom('p').hasStyle({ padding: '6px', margin: '12px' });
     });
 
     test('named arguments overrule option hash', async function(assert) {
-      await render(hbs`<p {{style (hash display="none") display="inline"}}></p>`);
+      await render(
+        hbs`<p {{style (hash display="none") display="inline"}}></p>`
+      );
 
-      assert.dom('p').hasStyle({ display: "inline" });
+      assert.dom('p').hasStyle({ display: 'inline' });
+    });
+
+    test('differently dasherized named arguments overrule option hash', async function(assert) {
+      await render(
+        hbs`<p {{style (hash font-size="10px" textAlign="left") fontSize="12px" text-align="right"}}></p>`
+      );
+
+      assert.dom('p').hasStyle({ fontSize: '12px', textAlign: 'right' });
+    });
+
+    test('supports multiple hashes', async function(assert) {
+      await render(
+        hbs`<p {{style (hash color="red" font-size="12px") (hash color="green")}}></p>`
+      );
+
+      assert.dom('p').hasStyle({ color: 'green', fontSize: '12px' });
```

Comment:
`color` isn't the best property to test against:

```js
// Result:
{
  "color": "rgb(0, 128, 0)",
  "fontSize": "12px"
}
```

Replacing `color` with `display` should fix the tests.